extensions/theme-carbon/themes/dark_carbon.json

```diff
@@ -56,7 +56,7 @@
 		"tab.activeBackground": "#212121",
         "tab.activeForeground": "#ffffff",
         "tab.inactiveBackground": "#444444",
-        "tab.inactiveForeground": "#888888",
+        "tab.inactiveForeground": "#b6b6b6",
```

Comment:
where did the new colors come from?

Reply:
@kburtram I worked with Smitha yesterday